examples/js/loaders/DRACOLoader.js

```diff
@@ -220,7 +220,7 @@ THREE.DRACOLoader.prototype = Object.assign( Object.create( THREE.Loader.prototy
 
 	},
 
-	_initDecoder: function () {
+	preload: function () {
```

Comment:
Maybe it would be better to have a second method: public `preload() : this` and private `_initDecoder() : Promise`. No other public APIs in three.js return Promises, so this would be a bit unusual otherwise. I don't think will users actually need to know when the preloading is complete.

Reply:
Yeah that seems pretty reasonable to me. Previously `getDecoderModule` returned a Promise so that's why I carried it over here, but keeping consistent with the rest of three.js is a smart move.

As for whether or not users would want to know when preloading has completed, I imagine the majority of use cases wouldn't need it. If we wanted to preserve that functionality we could add an optional onLoad callback parameter, but it may be better to wait until developers request it before adding such a feature.

Reply:
Just updated to include those changes

Reply:
Thanks!